homestar-runtime/src/cli/init.rs

```diff
@@ -0,0 +1,317 @@
+use std::{
+    fmt::Display,
+    fs::File,
+    io::{empty, stdout, Write},
+    path::PathBuf,
+    str::FromStr,
+};
+
```

Comment:
minor: we let cargo fmt sort imports, so remove the space and it should auto-format accordingly.